numpy/core/src/multiarray/nditer_api.c

```diff
@@ -2830,4 +2789,27 @@ npyiter_checkreducesize(NpyIter *iter, npy_intp count,
     }
     return count * (*reduce_innersize);
 }
+
+NPY_NO_EXPORT npy_bool
+npyiter_has_writeback(NpyIter *iter)
+{
+    int iop, nop;
+    npyiter_opitflags *op_itflags;
+    if (iter == NULL) {
+        return 0;
+    }
+    nop = NIT_NOP(iter);
+    op_itflags = NIT_OPITFLAGS(iter);
+    /* If NPY_OP_ITFLAG_HAS_WRITEBACK flag set on operand, resolve it.
+     * If the resolution fails (should never happen), continue from the
+     * next operand and discard the writeback scratch buffers, and return
+     * failure status
```

Comment:
This comment doesn't appear to reflect reality

Reply:
copy-pasted comment removed